packages/redux/src/raffles/selectors.ts

```diff
@@ -0,0 +1,295 @@
+import { createSelector } from 'reselect';
+import { buildQueryStringFromObject as generateRafflesHash } from '../helpers';
+import {
+  getAllRaffles as getAllRafflesFromReducer,
+  getEstimations,
+  getParticipationCreations,
+  getParticipations,
+  getRaffles as getRafflesFromReducer,
+} from './reducer';
+import { getEntities, getEntityById } from '../entities/selectors';
+import type { DetailedRaffle } from './types/raffles.types';
+import type {
+  Raffle,
+  RaffleParticipation,
+  RafflesQuery,
+} from '@farfetch/blackout-client';
+import type { RafflesState } from './types';
+import type { StoreState } from '../types/storeState.types';
+
+const getRaffleEntities = (state: StoreState) => getEntities(state, 'raffles');
+const getRaffleParticipationEntities = (state: StoreState) =>
+  getEntities(state, 'raffleParticipations');
+
+/**
+ * Returns all raffles from state.
+ *
+ * @param state - Application state
+ *
+ * @returns  All Raffle Participations entities in the state.
+ */
+const getRaffleParticipations: (
+  state: StoreState,
+) => RaffleParticipation[] | [] = createSelector(
```

Comment:
No need to have the `[]` type as it is included by the `RaffleParticipation[]`

```suggestion
) => RaffleParticipation[] = createSelector(
```

Check the remaining selectors that have this problem as well, please.

Reply:
removed. pls double check